.github/workflows/continuous-integration-workflow.yml

```diff
@@ -61,6 +61,21 @@ jobs:
       - name: "extract ct-ng"
         run: |
           tar -xf ct-ng.tar
+      - name: cache tarballs
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: src.tar
+          key: src.tar-${{ hashFiles('.local/share/crosstool-ng/packages') }}-${{ hashFiles('.local/share/crosstool-ng/samples') }}
+          restore-keys: |
+            src.tar-${{ hashFiles('.local/share/crosstool-ng/packages') }}-
+            src.tar-
+      - name: extract tarballs
+        if: ${{ steps.cache.outputs.cache-hit == 'true' }}
+        continue-on-error: true
+        run: |
+          tar -xvf src.tar
+          touch stamp
```

Comment:
Is this part still needed now. Won't the hashing on `packages`+`samples` be enough to ensure the cache gets dropped when we add a new version of a package?

Reply:
Yes because it can restore via restore-keys (backup-keys)